[2024-01-26T18:01:29.993-0300] {logging_mixin.py:151} INFO - Changing /mnt/c/Users/filipe.macedo_ifood/Documents/how-desafio-03/airflow/logs/dag_id=how-desafio-03/run_id=scheduled__2024-01-26T20:45:00+00:00/task_id=create_emr_cluster permission to 509
[2024-01-26T18:01:30.042-0300] {logging_mixin.py:151} INFO - Changing /mnt/c/Users/filipe.macedo_ifood/Documents/how-desafio-03/airflow/logs/dag_id=how-desafio-03/run_id=scheduled__2024-01-26T20:45:00+00:00/task_id=create_emr_cluster permission to 509
[2024-01-26T18:01:30.059-0300] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: how-desafio-03.create_emr_cluster scheduled__2024-01-26T20:45:00+00:00 [queued]>
[2024-01-26T18:01:30.069-0300] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: how-desafio-03.create_emr_cluster scheduled__2024-01-26T20:45:00+00:00 [queued]>
[2024-01-26T18:01:30.069-0300] {taskinstance.py:1361} INFO - Starting attempt 2 of 5
[2024-01-26T18:01:30.097-0300] {taskinstance.py:1382} INFO - Executing <Task(EmrCreateJobFlowOperator): create_emr_cluster> on 2024-01-26 20:45:00+00:00
[2024-01-26T18:01:30.099-0300] {standard_task_runner.py:57} INFO - Started process 1149 to run task
[2024-01-26T18:01:30.100-0300] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'how-desafio-03', 'create_emr_cluster', 'scheduled__2024-01-26T20:45:00+00:00', '--job-id', '283', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpi9zejjio']
[2024-01-26T18:01:30.101-0300] {standard_task_runner.py:85} INFO - Job 283: Subtask create_emr_cluster
[2024-01-26T18:01:30.161-0300] {logging_mixin.py:151} INFO - Changing /mnt/c/Users/filipe.macedo_ifood/Documents/how-desafio-03/airflow/logs/dag_id=how-desafio-03/run_id=scheduled__2024-01-26T20:45:00+00:00/task_id=create_emr_cluster permission to 509
[2024-01-26T18:01:30.163-0300] {task_command.py:416} INFO - Running <TaskInstance: how-desafio-03.create_emr_cluster scheduled__2024-01-26T20:45:00+00:00 [running]> on host 14111-NB.
[2024-01-26T18:01:30.264-0300] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Filipe' AIRFLOW_CTX_DAG_ID='how-desafio-03' AIRFLOW_CTX_TASK_ID='create_emr_cluster' AIRFLOW_CTX_EXECUTION_DATE='2024-01-26T20:45:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-26T20:45:00+00:00'
[2024-01-26T18:01:30.265-0300] {emr.py:742} INFO - Creating job flow using aws_conn_id: aws_default, emr_conn_id: emr_default
[2024-01-26T18:01:30.273-0300] {logging_mixin.py:151} WARNING - /usr/local/lib/python3.10/dist-packages/airflow/providers/amazon/aws/operators/emr.py:750 UserWarning: Unable to find Amazon Elastic MapReduce Connection ID 'emr_default', using an empty initial configuration. If you want to get rid of this warning message please provide a valid `emr_conn_id` or set it to None.
[2024-01-26T18:01:30.282-0300] {base.py:73} INFO - Using connection ID 'aws_default' for task execution.
[2024-01-26T18:01:30.283-0300] {connection_wrapper.py:340} INFO - AWS Connection (conn_id='aws_default', conn_type='aws') credentials retrieved from login and password.
[2024-01-26T18:01:31.382-0300] {emr.py:756} INFO - Job flow with id j-17P6U81DT1SYY created
[2024-01-26T18:01:31.689-0300] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=how-desafio-03, task_id=create_emr_cluster, execution_date=20240126T204500, start_date=20240126T210130, end_date=20240126T210131
[2024-01-26T18:01:31.723-0300] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-26T18:01:31.739-0300] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
